node/src/BaseClient.ts

```diff
@@ -185,6 +185,7 @@ export type ReturnType =
     | null
     | boolean
     | bigint
+    | Buffer
```

Comment:
remove?

Reply:
This is required to make the change in transaction testss